source-browser.html

```diff
@@ -181,13 +209,51 @@ <h1 id="loading" class="m1 center display-none">⌛</h1>
           .focus();
       }
 
+      function createNavbar(item) {
+          $('#li').append(
+              $('<a>', {
+                  id: 'li' + item,
+                  href: '#' + item,
+              }).text(item),
+          );
+      }
+
+      $(window).scroll(() => {
+          let top = $(window).scrollTop();
+          let bottom = $(window).scrollTop() + $(window).height();
+          for(let i = 0; i < TOPICS.length; i++){
```

Comment:
This should be possible with `forEach`. Don't create a new function, just use an anonymous arrow function.